repository/src/main/resources/db/changelog/db.changelog-rdm-2917.xml

```diff
@@ -0,0 +1,504 @@
+        <createTable tableName="case_field_acl">
+            <column name="id" type="serial">
+                <constraints nullable="false"/>
+            </column>
+            <column name="case_field_id" type="integer">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_role_id" type="integer">
+                <constraints nullable="true"/>
+            </column>
+            <column name="create" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="read" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="update" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="delete" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="live_from" type="date" >
+                <constraints nullable="true"/>
+            </column>
+            <column name="live_to" type="date" >
+                <constraints nullable="true"/>
+            </column>
+            <column name="created_at" type="datetime" defaultValue="now()">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addPrimaryKey columnNames="id"
+                       constraintName="pk_case_field_acl"
+                       tableName="case_field_acl"/>
+
+        <!-- Add foreign key for case_field_id column linking to case_field table -->
+        <addForeignKeyConstraint baseColumnNames="case_field_id"
+                                 baseTableName="case_field_acl"
+                                 constraintName="fk_case_field_acl_case_field_id_case_field_id"
+                                 referencedColumnNames="id"
+                                 referencedTableName="case_field"/>
+
+        <!-- Add foreign key for user_role_id column linking to user_role table -->
+        <addForeignKeyConstraint baseColumnNames="user_role_id"
+                                 baseTableName="case_field_acl"
+                                 constraintName="fk_case_field_acl_user_role_id_user_role_id"
+                                 referencedColumnNames="id"
+                                 referencedTableName="user_role"/>
+
+        <!-- Add unique constraint for case_field_id and user_role_id columns -->
+        <addUniqueConstraint columnNames="case_field_id, user_role_id"
+                             constraintName="unique_case_field_acl_case_field_id_user_role_id"
+                             tableName="case_field_acl"/>
+
+        <!-- copy table data to the new case_field_acl table -->
+        <sql>
+            insert into case_field_acl select * from case_field_user_role;
+        </sql>
+
+        <!-- Add new role_id column to the case_field_acl table -->
+        <addColumn tableName="case_field_acl">
```

Comment:
I don't see the benefit of splitting the creation of this column from the creation of the table it belongs to.